### PR TITLE
Added a redirection to the registration view in case the token is invalid

### PR DESCRIPTION
**Pull request by ccomb**
When using a login link with an invalid, we get a bare 403 error page. I added a redirect to the auth page.
Also removed dead code and fixed some translations

### CODE REVIEW

File: backend/authentication/locale/fr/LC_MESSAGES/django.po [hunk 1]
```diff
@@ -42,51 +42,52 @@ msgstr "Je m'engage à respecter les conditions d'utilisation"
...
 
-#: authentication/views.py:53 authentication/views.py:91
+#: authentication/views.py:41 authentication/views.py:86
 msgid "The link does not expire"
 msgstr "Le lien n'a pas de date d'expiration"
 
-#: authentication/views.py:63
+#: authentication/views.py:53
+#, fuzzy
+#| msgid "You seem already registered. Try to use the Connection form."
```
Comment (vjousse) [inline backend/authentication/locale/fr/LC_MESSAGES/django.po:68]: You should remove the fuzzy tag or the translation won't be compiled.

File: backend/authentication/locale/fr/LC_MESSAGES/django.po [hunk 2]
```diff
@@ -42,51 +42,52 @@ msgstr "Je m'engage à respecter les conditions d'utilisation"
...
-#: authentication/views.py:150
+#: authentication/views.py:109
 msgid "You must be authenticated to access this page"
 msgstr "Vous devez être authentifié(e) pour accéder à cette page"
 
+#~ msgid "The token has expired"
+#~ msgstr "Le jeton a expiré"
+
 #~ msgid "I undertake not to use the data for commercial use"
 #~ msgstr "Je m'engage à respecter les conditions d'utilisation"
```
Comment (vjousse) [inline backend/authentication/locale/fr/LC_MESSAGES/django.po:92]: Remove useless comments?

File: backend/backend/locale/fr/LC_MESSAGES/django.po
Comment (vjousse) [inline backend/backend/locale/fr/LC_MESSAGES/django.po:1]: Should not be included in the commit, nothing new here.

**Review by vjousse — APPROVED** [review]
Some little nitpicks otherwise, good job :+1:

File: backend/authentication/locale/fr/LC_MESSAGES/django.po (hunk 1 above)
Comment (ccomb) [reply backend/authentication/locale/fr/LC_MESSAGES/django.po:68]: :+1: I admit I don't really take time to manage the comments in the po file...

File: backend/backend/locale/fr/LC_MESSAGES/django.po
Comment (ccomb) [reply backend/backend/locale/fr/LC_MESSAGES/django.po:1]: It's an update in the line position, I let the default poedit output
